documentation/sections/get_set.md

```diff
@@ -0,0 +1,11 @@
+## get and set
+Get and set are intentionally not implemented as keywords in CoffeeScript.
+
+This is by design. While using get/set is still possible, CoffeeScript considers them anti-patterns.  
+
+In ECMAScript these convenience function decorators were introduced for very specific uses with the browser DOM.  
+
+If you are still convinced you need to use `get`/`set`, here are some workarounds:
```

Comment:
I think the documentation should avoid sounding judgmental. This section should probably be written more like:

`get` and `set`, as keywords preceding functions or class methods, are intentionally not implemented in CoffeeScript. This is to avoid grammatical ambiguity, since in CoffeeScript such a construct looks identical to a function call (e.g. `get(function foo() {})`) and because there is an alternate syntax that is slightly more verbose but just as effective:

> (insert the best `Object.defineProperty` example here)